wasm/js-canvas.go

```diff
@@ -25,24 +25,27 @@ func GetCanvas(domElementId string) Canvas {
 // Notes:
 // Consider drawImage, if we can generate the correct format to output in 1 call.
 // https://developer.mozilla.org/en-US/docs/Web/API/CanvasRenderingContext2D/drawImage
-// Also find out if we can do double buffering with canvas
+// OR
+// access the canvas pixel data directly might be faster - at least for unscaled output
+//  var id = ctx.getImageData(0, 0, canvasWidth, canvasHeight)
+//  var pixels = id.data
+//  pixels[0] = 0
+//  pixels[1] = 255
+//  pixels[2] = 255
+//  pixels[3] = 255
 
 func (c Canvas) SetColor(color anotherworld.Color) {
-	c.fillStyle(getHexColorFrom(color))
+	c.context2d.Set("fillStyle", fmt.Sprintf("rgb(%d, %d, %d)", color.R, color.G, color.B))
 }
 
 func (c Canvas) DrawPoint(x, y int) {
-	c.fillRect(x, y, 1, 1)
+	c.context2d.Call("fillRect", x, y, 1, 1)
 }
 
 func (c Canvas) fillStyle(style string) {
 	c.context2d.Set("fillStyle", style)
 }
 
-func (c Canvas) fillRect(x, y, length, width int) {
-	c.context2d.Call("fillRect", x, y, length, width)
-}
-
-func getHexColorFrom(color anotherworld.Color) string {
-	return fmt.Sprintf("#%X%X%X", color.R, color.G, color.B)
+func (c Canvas) fillRect(x, y, width, height int) {
+	c.context2d.Call("fillRect", x, y, width, height)
```

Comment:
if you're calling `c.context2d.Call("fillRect", x, y, 1, 1)` directly above is this function still needed?